### ISSUE TITLE
[branch: indexingNewData] bug list

### ISSUE THREAD

**Issue by martingrimmer** [IIDP/OSTMap#25]
currently there are some bugs:
- it writes its results into a table with name TestData
- it does not write into RawTwitterData
  - it should write the original data to RawTwitterData AND the term index to TermIndex
- the are entries in the result table with an emtpy token
- it contains terms with a single sign, for example: `( text:\x00\x00\x00\x00W(\xA4v\xB4~\xF5\xCD []`
- is something like `(@ text:\x00\x00\x00\x00W(\xA3\x9B\x93#Oo []` or `(hourly) text:\x00\x00\x00\x00W(\xA3\xC2\x0E\xDD\x0F4 []` ok?
  - do we want sign like `()[]/\<>*+'"` and so on in terms?
- currently it does not transform the input into lower case!
  - `Eliza Waldorf user:\x00\x00\x00\x00W(\xA4\x0A\xFD\xE6t\x93 []`
- the code quality
  - not correct author tags  -> remove all author tags from code
  - wrong comments
  - no comments
- the tokenizer from commons library should be used
- there is no value (count of a token per tweet)

some example lines from the result:

```
)))) text:\x00\x00\x00\x00W(\xA4\x95\xF5\x88\xBA_ []
)))) text:\x00\x00\x00\x00W(\xA4\xB3\xBC\xC4\xD8\xB3 []
)))))))))))) text:\x00\x00\x00\x00W(\xA3\xAF\xFB\xBE\xCC} []
))))))ahahaaaa text:\x00\x00\x00\x00W(\xA3\xB3k\x0B\xB8\x9B []
)))00))\\u0442\\u044b text:\x00\x00\x00\x00W(\xA4k\x05\xE9\xD5\x96 []
)\\ text:\x00\x00\x00\x00W(\xA4\x09\xC5\xF9\xB8\x97 []
)\\n\\n#jag\\u00c4lskarmittjobb text:\x00\x00\x00\x00W(\xA4R\x9C\xE5\x92\xEA []
)\\n\\njag\\u00c4lskarmittjobb text:\x00\x00\x00\x00W(\xA4R\x9C\xE5\x92\xEA []
)\\nhttps\\/\\/tco\\/vahgzwa1cz text:\x00\x00\x00\x00W(\xA4t-\xCF\xBEC []
* text:\x00\x00\x00\x00W(\xA3\xD9\xA5\x00\xF0\x1A []
* text:\x00\x00\x00\x00W(\xA4i\x07\\\xE66 []
***** text:\x00\x00\x00\x00W(\xA4\x0Bd\xAE\xAAx []
*Daddy's Girl* user:\x00\x00\x00\x00W(\xA3\xEE\xBA\xD0\xA9\x06 []
*\\nhave text:\x00\x00\x00\x00W(\xA4\x03o|\x1F, []
*\\nhave text:\x00\x00\x00\x00W(\xA46\x9C\xFEm\xE0 []
*\\nhave text:\x00\x00\x00\x00W(\xA4V\xD4\x0B'\xED []
*\\nhave text:\x00\x00\x00\x00W(\xA4~\xBF}r\xCD []
*^* text:\x00\x00\x00\x00W(\xA4U\xCC\xE9\x8BQ []
*encara text:\x00\x00\x00\x00W(\xA4\x82\xAB\xC8R\x01 []
*forever text:\x00\x00\x00\x00W(\xA3\xB1`\x89\x9A\xB6 []
*forever text:\x00\x00\x00\x00W(\xA3\xDAqZa\xFB []
*la text:\x00\x00\x00\x00W(\xA4$\xD0\xB7\xEA\x8A []
*ohhhh text:\x00\x00\x00\x00W(\xA4F\xDE<\x90\xCE []
*on text:\x00\x00\x00\x00W(\xA3\xF1\xD3\xF5\x95\xD7 []
*you* text:\x00\x00\x00\x00W(\xA4\x1E\xF9\xA4O\x0A []
+ text:\x00\x00\x00\x00W(\xA3\xA8f\xEDJ\x1D []
+ text:\x00\x00\x00\x00W(\xA3\xC3\xF9\x04\xE5L []
```

**Comment by HansPog** [comment]
also the unit tests are in the wrong package

**Comment by loewenzahm** [comment]
"the are entries in the result table with an emtpy token"
"it contains terms with a single sign, ..."
 "do we want sign like ()[]/<>*+'" and so on in terms?"
--> IMHO these problems result from tokenizer. That is why I opened a new ticket assigned to @CSchott 

"the tokenizer from commons library should be used"
--> @CSchott  's task, opened new ticket 

"there is no value (count of a token per tweet)"
--> @martingrimmer , you told us to use empty value in case the token occurs only once.

**Comment by loewenzahm** [comment]
I fixed three of the bugs (see above) and created two new tickets for @CSchott because imho the concerning problems do not result from my part of the code.

**Comment by HansPog** [comment]
I think there is a new version of the tokenizer (see #21). maybe you should merge master branch into yours? (the package of the tokenizer has changed so you have to adapt your code a little)

**Comment by HansPog** [comment]
there is still a folder "twitterStreamIndexing" on your branch. if this contains something useful it has to be adapted to new project structure, else it should be deleted

**Comment by loewenzahm** [comment]
I think your first comment should be placed in #28 .

I deleted the old files/folder. So next try to close issue.
